apps/omg/test/omg/conformance/signature_test.exs

```diff
@@ -109,4 +109,27 @@ defmodule OMG.Conformance.SignatureTest do
 
     assert solidity_hash == OMG.TypedDataHash.hash_struct(tx)
   end
+
+  defp parse_contracts() do
```

Comment:
I'd leave a comment to a new reader of this. There used to be `taken from the `plasma-contracts` deployment snapshot` there. A single sentence justifying this operation.

Oh, and I'd leave a hint that there might be several implementations of `parse_contracts`, so that someone who goes about modifying this bit of code doesn't create a fork by accident. (or actually just put it somewhere under `OMG.Eth.SomeModule`, I think this is justified)